FIX public ticket layout for Safari on macOS (v18)

W1W1-M opened the pull request:
# FIX public ticket layout for Safari on macOS (v18)
This is a backport for #33728 & #33742

W1W1-M committed:
Removed redundant 'centpercent' class from ticket layouts to fix Safari rendering issues on macOS

W1W1-M committed:
Fixed public ticket layout. Removed undeclared CSS class.

W1W1-M committed:
Fixed ticket public list layout for Safari

eldy commented:
Can you check this PR instead
https://github.com/Dolibarr/dolibarr/commit/d77898ff3dba5e7f968e7a097ded4b0d7c7cc343

W1W1-M commented:
@eldy Retested this PR on Safari, MS Edge/Chrome & Firefox. Index, view, list & create layouts are centered and taking up normal width.

eldy commented:
> @eldy Retested this PR on Safari, MS Edge/Chrome & Firefox. Index, view, list & create layouts are centered and taking up normal width.

And on smartphone ?

W1W1-M committed:
Revert "Fixed public ticket layout. Removed undeclared CSS class."

This reverts commit 2b8f7e0f1b1e1904f7d1195805ad2ad9c3689675.

W1W1-M committed:
Revert "Fixed ticket public list layout for Safari"

This reverts commit be89d91acaa79011adc9b4c6ee9a67c4ed8ea9e7.

W1W1-M committed:
Revert "Removed redundant 'centpercent' class from ticket layouts to fix Safari rendering issues on macOS"

This reverts commit 4759cab48cb960d762a59ac09c70a880d05d3e45.

W1W1-M committed:
Added Safari to browser list of flexcontainer

W1W1-M commented:
> > @eldy Retested this PR on Safari, MS Edge/Chrome & Firefox. Index, view, list & create layouts are centered and taking up normal width.
> 
> And on smartphone ? And what about my suggested fix ?

Removing `flexcontainer` doesn't help for Safari and seems to also break things for chrome and firefox. I looked closer at the `flexcontainer` CSS class and noticed Safari was missing from list of browsers used for `display: inline-flex`. 

I've reverted my older fixes and just added Safari to list of browser for `flexcontainer` and everything is rendering correctly.

As for iPad & iPhone, I still have the problem. I propose to make a separate PR for this.

W1W1-M commented:
Better fix #33933
